.travis.yml

```diff
@@ -174,6 +175,7 @@ script:
   - python -m pytest -v --durations=10 test/cython_test.py
   - python -m pytest -v --durations=10 test/credis_test.py
   - python -m pytest -v --durations=10 test/node_manager_test.py
+  - python -m pytest -v --durations=10 test/object_manager_test.py
```

Comment:
There might not be enough memory/CPUs on Travis to run this test. That's why I put it in Jenkins instead.

Reply:
Why move it here instead of Jenkins?

Reply:
I see. Because this test fails frequently in Jenkins, but it looks like it works well in Travis...

Reply:
Weird. I wanted this test to be pretty stressful, e.g., to require lots of memory/CPUs, which is why Jenkins should be a good fit. It's fine to move it here if it's passing though eventually I may try to move it back to Jenkins (provided I can get it to pass).

Reply:
Add your name after TODO? so people would know who to ask when they see this in the future. 
Also, it'd be nice to explain why this test should be put in Jenkins, e.g.
`object_manager_test requires a lot of CPU/memory, and better be put in Jenkins. However, it fails frequently in Jenkins, but works well in Travis. We should consider moving it back to Jenkins once we figure out the reason.`